wandb: Currently logged in as: xijohnny9 (ps-matching). Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.15.12
wandb: Run data is saved locally in checkpoints/balls/wandb/run-20231020_152758-ploy9ux1
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ruby-elevator-22
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ps-matching/lightning_logs
wandb: üöÄ View run at https://wandb.ai/ps-matching/lightning_logs/runs/ploy9ux1
/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/torch/nn/modules/lazy.py:180: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
You are using a CUDA device ('NVIDIA A100-SXM4-80GB') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
It.  |Err         
-------------------
    0|1.741301e-03|
   10|4.505656e-04|
   20|9.216731e-05|
   30|7.902615e-05|
   40|1.275345e-05|
   50|1.278493e-06|
   60|1.213388e-07|
   70|1.145560e-08|
   80|1.081099e-09|
   90|1.020281e-10|
scot took 0.8329400101210922 seconds on label 0 with 270 samples
It.  |Err         
-------------------
    0|2.809079e-03|
   10|5.499367e-04|
   20|1.443665e-04|
   30|8.735253e-06|
   40|8.545005e-07|
   50|1.792380e-07|
   60|4.790884e-08|
   70|1.351425e-08|
   80|3.883175e-09|
   90|1.123493e-09|
  100|3.258566e-10|
scot took 0.7697484130039811 seconds on label 1 with 203 samples
It.  |Err         
-------------------
    0|2.708724e-02|
   10|7.310166e-03|
   20|2.396687e-03|
   30|1.124125e-03|
   40|9.039148e-04|
   50|3.894564e-04|
   60|8.851269e-05|
   70|1.849510e-05|
   80|4.176539e-06|
   90|1.004379e-06|
  100|2.488089e-07|
  110|6.233685e-08|
  120|1.567708e-08|
  130|3.946837e-09|
  140|9.938313e-10|
scot took 2.595076890895143 seconds on label 2 with 41 samples
It.  |Err         
-------------------
    0|3.323766e-04|
   10|3.897030e-06|
   20|2.448890e-08|
   30|1.495317e-10|
scot took 39.11733963992447 seconds on label 3 with 2494 samples
It.  |Err         
-------------------
    0|1.507963e-03|
   10|8.122161e-05|
   20|3.161555e-06|
   30|1.278478e-07|
   40|5.366418e-09|
   50|2.315858e-10|
scot took 0.700871234992519 seconds on label 4 with 362 samples
It.  |Err         
-------------------
    0|2.776296e-04|
   10|1.114386e-05|
   20|9.493385e-07|
   30|2.103822e-07|
   40|4.701331e-08|
   50|1.048113e-08|
   60|2.335340e-09|
   70|5.202890e-10|
scot took 51.014945794129744 seconds on label 5 with 2147 samples
It.  |Err         
-------------------
    0|1.277898e-03|
   10|2.112459e-04|
   20|3.873565e-05|
   30|8.183905e-06|
   40|2.345902e-06|
   50|7.160186e-07|
   60|2.202070e-07|
   70|6.778066e-08|
   80|2.086456e-08|
   90|6.422556e-09|
  100|1.976980e-09|
  110|6.085478e-10|
scot took 0.9950796628836542 seconds on label 6 with 297 samples
It.  |Err         
-------------------
    0|1.179346e-03|
   10|1.186837e-04|
   20|1.706941e-05|
   30|3.919883e-07|
   40|8.421995e-09|
   50|1.811713e-10|
scot took 1.1418594890274107 seconds on label 7 with 426 samples
It.  |Err         
-------------------
    0|3.224535e-03|
   10|7.028256e-04|
   20|1.536405e-06|
   30|7.946576e-08|
   40|4.841589e-09|
   50|2.951667e-10|
scot took 0.5387439369224012 seconds on label 8 with 190 samples
It.  |Err         
-------------------
    0|2.471680e-03|
   10|1.181758e-04|
   20|1.844124e-05|
   30|2.893843e-06|
   40|4.501468e-07|
   50|6.975655e-08|
   60|1.079287e-08|
   70|1.668920e-09|
   80|2.580142e-10|
scot took 1.032657773932442 seconds on label 9 with 228 samples
It.  |Err         
-------------------
    0|2.447340e-03|
   10|4.055295e-05|
   20|1.289411e-07|
   30|3.910662e-10|
scot took 0.5397606969345361 seconds on label 10 with 249 samples
It.  |Err         
-------------------
    0|3.048753e-03|
   10|3.028507e-07|
   20|7.196819e-10|
scot took 1.1950503028929234 seconds on label 11 with 390 samples
It.  |Err         
-------------------
    0|1.398382e-03|
   10|1.589212e-06|
   20|7.589311e-10|
scot took 0.7631090309005231 seconds on label 12 with 434 samples
It.  |Err         
-------------------
    0|4.982770e-04|
   10|2.707058e-05|
   20|3.112706e-06|
   30|4.800081e-07|
   40|8.030802e-08|
   50|1.351070e-08|
   60|2.268003e-09|
   70|3.801520e-10|
scot took 6.351188261993229 seconds on label 13 with 1040 samples
It.  |Err         
-------------------
    0|5.888322e-02|
   10|2.618675e-03|
   20|7.485301e-05|
   30|3.336241e-06|
   40|1.502387e-07|
   50|6.771544e-09|
   60|3.052490e-10|
scot took 1.2602095280308276 seconds on label 14 with 18 samples
It.  |Err         
-------------------
    0|2.396312e-04|
   10|3.092248e-05|
   20|1.629310e-06|
   30|1.193349e-07|
   40|1.023475e-08|
   50|9.194022e-10|
scot took 83.06224899506196 seconds on label 15 with 2500 samples
It.  |Err         
-------------------
    0|6.412288e-04|
   10|3.364498e-05|
   20|1.455993e-06|
   30|6.973890e-08|
   40|3.324170e-09|
   50|1.584126e-10|
scot took 9.831237311009318 seconds on label 16 with 1082 samples
It.  |Err         
-------------------
    0|3.903871e-04|
   10|2.594494e-06|
   20|3.484134e-08|
   30|5.298361e-10|
scot took 14.471987782046199 seconds on label 17 with 1658 samples
It.  |Err         
-------------------
    0|8.601713e-04|
   10|5.628190e-05|
   20|7.780060e-08|
   30|1.103487e-10|
scot took 2.752843288006261 seconds on label 18 with 678 samples
It.  |Err         
-------------------
    0|1.327069e-03|
   10|3.536699e-06|
   20|3.971416e-08|
   30|1.135116e-09|
   40|3.304822e-11|
scot took 1.98932003416121 seconds on label 19 with 589 samples
It.  |Err         
-------------------
    0|1.623727e-02|
   10|4.365393e-03|
   20|5.482602e-04|
   30|1.616897e-04|
   40|5.488688e-05|
   50|1.905637e-05|
   60|6.824628e-06|
   70|2.512173e-06|
   80|9.455647e-07|
   90|3.625717e-07|
  100|1.412274e-07|
  110|5.573620e-08|
  120|2.223202e-08|
  130|8.942722e-09|
  140|3.620537e-09|
  150|1.473010e-09|
  160|6.014948e-10|
scot took 0.9087891101371497 seconds on label 20 with 62 samples
It.  |Err         
-------------------
    0|2.781333e-03|
   10|3.562148e-04|
   20|2.047593e-04|
   30|2.225116e-05|
   40|2.694881e-06|
   50|3.266671e-07|
   60|3.959367e-08|
   70|4.799113e-09|
   80|5.817126e-10|
scot took 0.4899412931408733 seconds on label 21 with 178 samples
It.  |Err         
-------------------
    0|1.186942e-03|
   10|2.286627e-04|
   20|1.043614e-04|
   30|6.981891e-05|
   40|3.846879e-05|
   50|1.599159e-05|
   60|5.947320e-06|
   70|2.129387e-06|
   80|7.525251e-07|
   90|2.647263e-07|
  100|9.297493e-08|
  110|3.263487e-08|
  120|1.145269e-08|
  130|4.018844e-09|
  140|1.410208e-09|
  150|4.948355e-10|
scot took 2.8633110669907182 seconds on label 22 with 463 samples
It.  |Err         
-------------------
    0|2.444928e-03|
   10|2.328882e-04|
   20|8.448682e-05|
   30|7.322738e-06|
   40|1.126450e-06|
   50|1.992379e-07|
   60|3.752490e-08|
   70|7.223778e-09|
   80|1.400270e-09|
   90|2.720240e-10|
scot took 0.4907525759190321 seconds on label 23 with 219 samples
It.  |Err         
-------------------
    0|3.047072e-03|
   10|6.488680e-06|
   20|4.291471e-08|
   30|2.895015e-10|
scot took 0.6689027340617031 seconds on label 24 with 218 samples
It.  |Err         
-------------------
    0|2.895326e-04|
   10|8.523333e-06|
   20|9.458211e-09|
   30|1.486926e-11|
scot took 16.899314810987562 seconds on label 25 with 1828 samples
It.  |Err         
-------------------
    0|8.096506e-04|
   10|2.856305e-05|
   20|9.774401e-06|
   30|1.444052e-06|
   40|1.852145e-07|
   50|2.341044e-08|
   60|2.955246e-09|
   70|3.730386e-10|
scot took 4.643905200995505 seconds on label 26 with 841 samples
It.  |Err         
-------------------
    0|4.642198e-04|
   10|1.343612e-06|
   20|6.264750e-09|
   30|2.997475e-11|
scot took 7.10598425893113 seconds on label 27 with 1232 samples
It.  |Err         
-------------------
    0|1.267612e-03|
   10|1.237481e-05|
   20|6.720016e-06|
   30|3.461317e-06|
   40|1.622472e-06|
   50|7.244707e-07|
   60|3.163068e-07|
   70|1.367326e-07|
   80|5.885104e-08|
   90|2.528277e-08|
  100|1.085291e-08|
  110|4.657125e-09|
  120|1.998137e-09|
  130|8.572451e-10|
scot took 4.281114764045924 seconds on label 28 with 602 samples
It.  |Err         
-------------------
    0|1.187671e-03|
   10|9.731796e-07|
   20|1.790069e-09|
   30|3.627688e-12|
scot took 1.206239994848147 seconds on label 29 with 576 samples
It.  |Err         
-------------------
    0|6.771234e-03|
   10|1.137350e-04|
   20|3.462546e-06|
   30|1.226001e-07|
   40|4.334366e-09|
   50|1.532031e-10|
scot took 1.0609093359671533 seconds on label 30 with 113 samples
It.  |Err         
-------------------
    0|8.571398e-03|
   10|1.229431e-03|
   20|2.598717e-05|
   30|3.090880e-06|
   40|3.697565e-07|
   50|4.425988e-08|
   60|5.298358e-09|
   70|6.342751e-10|
scot took 0.8116847879718989 seconds on label 31 with 98 samples
/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/ot/bregman.py:536: UserWarning: Sinkhorn did not converge. You might want to increase the number of iterations `numItermax` or the regularization parameter `reg`.
  warnings.warn("Sinkhorn did not converge. You might want to "
/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/ot/bregman.py:511: UserWarning: Warning: numerical errors at iteration 0
  warnings.warn('Warning: numerical errors at iteration %d' % ii)
It.  |Err         
-------------------
    0|1.109768e-02|
   10|3.054105e-05|
   20|1.540929e-07|
   30|7.750275e-10|
scot took 0.3998133901041001 seconds on label 32 with 99 samples
It.  |Err         
-------------------
    0|1.681563e-02|
   10|2.613184e-03|
   20|7.411689e-04|
   30|9.065924e-05|
   40|1.135834e-05|
   50|1.572560e-06|
   60|5.146341e-07|
   70|3.342969e-07|
   80|2.298802e-07|
   90|1.583666e-07|
  100|1.090955e-07|
  110|7.514824e-08|
  120|5.176169e-08|
  130|3.565191e-08|
  140|2.455537e-08|
  150|1.691231e-08|
  160|1.164808e-08|
  170|8.022363e-09|
  180|5.525199e-09|
  190|3.805326e-09|
It.  |Err         
-------------------
  200|2.620804e-09|
  210|1.804998e-09|
  220|1.243135e-09|
  230|8.561685e-10|
scot took 7.400545543991029 seconds on label 33 with 50 samples
It.  |Err         
-------------------
    0|1.814237e-03|
   10|2.897801e-05|
   20|2.015408e-06|
   30|1.526886e-07|
   40|1.152078e-08|
   50|8.689546e-10|
scot took 2.021219056798145 seconds on label 34 with 396 samples
It.  |Err         
-------------------
    0|4.702042e-04|
   10|3.034958e-06|
   20|1.051896e-08|
   30|4.225567e-11|
scot took 16.555117617128417 seconds on label 35 with 1755 samples
It.  |Err         
-------------------
    0|     nan|
CONVERGENCE ERROR: Optimization procedure runs into numerical errors with the hyperparameters specified. Please try aligning with higher values of epsilon.
Traceback (most recent call last):
  File "/rxrx/scratch/sandbox/johnny.xi/matching/main.py", line 41, in <module>
    trainer.fit(model = model, datamodule = data)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 545, in fit
    call._call_and_handle_interrupt(
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 44, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 581, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 951, in _run
    call._call_setup_hook(self)  # allow user to setup lightning_module in accelerator environment
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 93, in _call_setup_hook
    _call_callback_hooks(trainer, "setup", stage=fn)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 208, in _call_callback_hooks
    fn(trainer, trainer.lightning_module, *args, **kwargs)
  File "/rxrx/scratch/sandbox/johnny.xi/matching/matching/callbacks.py", line 39, in setup
    scot_coupling = scot_matching(x1_lab, x2_lab)
  File "/rxrx/scratch/sandbox/johnny.xi/matching/matching/utils.py", line 100, in scot_matching
    _, _ = scot.align()
TypeError: cannot unpack non-iterable NoneType object
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: 
wandb: Run summary:
wandb:  SCOT Trace 0 1.20207
wandb:  SCOT Trace 1 0.59042
wandb: SCOT Trace 10 1.01227
wandb: SCOT Trace 11 2.52886
wandb: SCOT Trace 12 0.58591
wandb: SCOT Trace 13 1.0292
wandb: SCOT Trace 14 1.02715
wandb: SCOT Trace 15 0.73664
wandb: SCOT Trace 16 0.60953
wandb: SCOT Trace 17 1.16071
wandb: SCOT Trace 18 3.6084
wandb: SCOT Trace 19 1.84738
wandb:  SCOT Trace 2 2.1994
wandb: SCOT Trace 20 0.35155
wandb: SCOT Trace 21 1.28103
wandb: SCOT Trace 22 0.99902
wandb: SCOT Trace 23 0.40527
wandb: SCOT Trace 24 1.96572
wandb: SCOT Trace 25 1.69068
wandb: SCOT Trace 26 1.41444
wandb: SCOT Trace 27 1.0488
wandb: SCOT Trace 28 1.56618
wandb: SCOT Trace 29 0.82803
wandb:  SCOT Trace 3 0.56702
wandb: SCOT Trace 30 0.08758
wandb: SCOT Trace 31 1.69887
wandb: SCOT Trace 32 1.01702
wandb: SCOT Trace 33 2.66049
wandb: SCOT Trace 34 2.76063
wandb: SCOT Trace 35 0.94436
wandb:  SCOT Trace 4 0.51703
wandb:  SCOT Trace 5 2.24506
wandb:  SCOT Trace 6 1.58011
wandb:  SCOT Trace 7 1.57165
wandb:  SCOT Trace 8 1.87827
wandb:  SCOT Trace 9 1.77846
wandb: 
wandb: üöÄ View run ruby-elevator-22 at: https://wandb.ai/ps-matching/lightning_logs/runs/ploy9ux1
wandb: Ô∏è‚ö° View job at https://wandb.ai/ps-matching/lightning_logs/jobs/QXJ0aWZhY3RDb2xsZWN0aW9uOjEwODQ0OTkzNQ==/version_details/v7
wandb: Synced 5 W&B file(s), 0 media file(s), 2 artifact file(s) and 0 other file(s)
wandb: Find logs at: checkpoints/balls/wandb/run-20231020_152758-ploy9ux1/logs
Exception in thread IntMsgThr:
Traceback (most recent call last):
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/3.9.7/lib/python3.9/threading.py", line 973, in _bootstrap_inner
Exception in thread ChkStopThr:
Traceback (most recent call last):
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/3.9.7/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/3.9.7/lib/python3.9/threading.py", line 910, in run
    self.run()
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/3.9.7/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 285, in check_stop_status
    self._target(*self._args, **self._kwargs)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 299, in check_internal_messages
        self._loop_check_status(self._loop_check_status(

  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 223, in _loop_check_status
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 223, in _loop_check_status
    local_handle = request()
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 727, in deliver_stop_status
    local_handle = request()
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 743, in deliver_internal_messages
    return self._deliver_stop_status(status)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 459, in _deliver_stop_status
    return self._deliver_internal_messages(internal_message)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 481, in _deliver_internal_messages
    return self._deliver_record(record)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 428, in _deliver_record
    return self._deliver_record(record)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 428, in _deliver_record
    handle = mailbox._deliver_record(record, interface=self)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
    handle = mailbox._deliver_record(record, interface=self)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
    interface._publish(record)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    interface._publish(record)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self._sock_client.send_record_publish(record)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self.send_server_request(server_req)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._send_message(msg)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    self._sendall_with_error_handle(header + data)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
srun: error: dgx8: task 0: Exited with exit code 1
wandb: Currently logged in as: xijohnny9 (ps-matching). Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.15.12
wandb: Run data is saved locally in checkpoints/balls/wandb/run-20231020_153413-2j4cz9uc
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run laced-sun-23
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ps-matching/lightning_logs
wandb: üöÄ View run at https://wandb.ai/ps-matching/lightning_logs/runs/2j4cz9uc
/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet18_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet18_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
You are using a CUDA device ('NVIDIA A100-SXM4-80GB') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
It.  |Err         
-------------------
    0|2.492622e-04|
   10|1.407174e-04|
   20|1.809987e-04|
   30|1.022554e-04|
   40|8.397427e-05|
   50|2.023891e-05|
   60|1.190039e-05|
   70|7.423200e-06|
   80|4.631235e-06|
   90|2.887085e-06|
  100|1.798815e-06|
  110|1.120365e-06|
  120|6.976402e-07|
  130|4.343478e-07|
  140|2.703966e-07|
  150|1.683209e-07|
  160|1.047750e-07|
  170|6.521784e-08|
  180|4.059464e-08|
  190|2.526776e-08|
It.  |Err         
-------------------
  200|1.572759e-08|
  210|9.789397e-09|
  220|6.093246e-09|
  230|3.792634e-09|
  240|2.360656e-09|
  250|1.469347e-09|
  260|9.145672e-10|
scot took 33.42807340901345 seconds on label 0 with 825 samples
Traceback (most recent call last):
  File "/rxrx/scratch/sandbox/johnny.xi/matching/main.py", line 41, in <module>
    trainer.fit(model = model, datamodule = data)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 545, in fit
    call._call_and_handle_interrupt(
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 44, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 581, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 951, in _run
    call._call_setup_hook(self)  # allow user to setup lightning_module in accelerator environment
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 93, in _call_setup_hook
    _call_callback_hooks(trainer, "setup", stage=fn)
  File "/mnt/ps/home/CORP/johnny.xi/.pyenv/versions/matching/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 208, in _call_callback_hooks
    fn(trainer, trainer.lightning_module, *args, **kwargs)
  File "/rxrx/scratch/sandbox/johnny.xi/matching/matching/callbacks.py", line 47, in setup
    mse = latent_matching_score(scot_coupling, z_subset)
  File "/rxrx/scratch/sandbox/johnny.xi/matching/matching/utils.py", line 116, in latent_matching_score
    z_matched = coupling @ z   ### (n x n) x (n x 4)
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 1651 is different from 825)
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: 
wandb: Run summary:
wandb: SCOT Trace 0 4.8831
wandb: 
wandb: üöÄ View run laced-sun-23 at: https://wandb.ai/ps-matching/lightning_logs/runs/2j4cz9uc
wandb: Ô∏è‚ö° View job at https://wandb.ai/ps-matching/lightning_logs/jobs/QXJ0aWZhY3RDb2xsZWN0aW9uOjEwODQ0OTkzNQ==/version_details/v8
wandb: Synced 5 W&B file(s), 0 media file(s), 2 artifact file(s) and 0 other file(s)
wandb: Find logs at: checkpoints/balls/wandb/run-20231020_153413-2j4cz9uc/logs
srun: error: dgx8: task 0: Exited with exit code 1
